packages/docusaurus-theme-redoc/src/types/common.ts

```diff
@@ -5,6 +5,7 @@ import type { RecursivePartial } from './util';
 export type RedocProps = {
   spec?: Record<string, unknown>;
   specUrl?: string;
+  onLoaded: (e?: Error) => void;
 };
```

Comment:
@dlcognite What do you think about just extending the type to include everything and then passing the props as `...rest`?

Something like:

```suggestion
export interface RedocProps extends Omit<RedocStandaloneProps, 'spec' | 'specUrl' | 'options'> {
  spec?: Record<string, unknown>;
  specUrl?: string;
};
```